provision.sh

```diff
@@ -3,7 +3,7 @@
 # Some dependencies
 sudo apt-get update
 sudo apt-get install -y libjudy-dev make g++ gdb git python-dev python-pip
-sudo pip install cython
+sudo pip install cython==0.22.1
```

Comment:
Because the tests (and, presumably, working code) hang when we use anything later than `frozen` version.